provisioning_templates/provision/kickstart_default.erb

```diff
@@ -149,10 +149,10 @@ skipx
 
   # VLAN (only on physical is recognized)
   if iface.virtual? && iface.tag.present? && iface.attached_to.present?
-    if rhel_compatible && os_major >= 6
-      network_options.push("--vlanid=#{iface.tag}")
-    elsif rhel_compatible && os_major >= 7
+    if rhel_compatible && os_major >= 7
```

Comment:
previously this was also added for fedora >=17, should it still be done?

Reply:
True, let me rethink this.